sparkmagic/sparkmagic/utils/configuration.py

```diff
@@ -164,6 +164,9 @@ def resource_limit_mitigation_suggestion():
 def ignore_ssl_errors():
     return False
 
+@_with_override
+def custom_certfiles_path():
+    return None
```

Comment:
@aggFTW this is how we handle None value for `custom_certfiles_path` , so I'm not sure what to do for ` example_config.json`

Reply:
If you did:

`"custom_certfiles_path": null,`

would it end up returning `None`?

I think it would. Maybe try that out.

Reply:
What do you think about not using ` "custom_certfiles_path": null` in the configuration file but telling in the readme that we can set this value ? 
I, personally,  don't like to have null value set somewhere

Reply:
The purpose of having an example_config.json in the repo is to have a place that lists all the configurable options for the library and show what the default values are, so this option would need to be there and match the python `None` value. Let's add it so that it translates to `None` if used.

Reply:
with ` json.loads(path)` null translates to None

https://docs.python.org/3/library/json.html#encoders-and-decoders